client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -216,17 +216,17 @@ const RecurringPaymentsPlanAddEditModal = ( {
 			<>
 				<p>
 					{ product
-						? translate( 'Edit your existing Recurring Payments plan.' )
+						? translate( 'Edit your existing payment plan.' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.